src/bin/pgcopydb/pgcmd.c

```diff
@@ -364,7 +363,7 @@ pg_dump_db(PostgresPaths *pgPaths,
 	if (!IS_EMPTY_STRING_BUFFER(safeURI.password))
 	{
 		if (pgpassword_found_in_env &&
-			!get_env_copy("PGPASSWORD", PGPASSWORD, MAXCONNINFO))
+			!get_env_dup("PGPASSWORD", &(PGPASSWORD)))
```

Comment:
Nitpicking: no need for parentheses here. Adding them doesn't change anything implementation-wide, but makes the code a little harder to read (for my eyes at least).